src/components/addtodo/AddTodo.js

```diff
@@ -43,8 +57,16 @@ function AddTodo() {
             type="button"
             value="Add"
             onClick={addHandler} />
+        <div className="dropdown">
+            <img alt="" className="avatar" src={data.image} />
```

Comment:
always try to add a meaning full data in alt attribute if we dont have proper data then in that case we can give a generic info like profile picture

Reply:
done